frontend/pages/admin/IntegrationsPage/cards/IdentityProviders/components/EndUserAuthSection/EndUserAuthSection.tsx

```diff
@@ -87,23 +89,27 @@ const IdpSection = () => {
         if (ae.status === 422) {
           renderFlash(
             "error",
-            `Couldn’t update: ${expandErrorReasonRequired(err)}.`
+            `Couldn't update: ${expandErrorReasonRequired(err)}.`
           );
           return;
         }
-        renderFlash("error", "Couldn’t update. Please try again.");
+        renderFlash("error", "Couldn't update. Please try again.");
```

Comment:
Do we have an official stance on curly quotes?  I've been using `&rsquo;` wherever possible.

Reply:
i dont think we have an official stance but Its rare that I've seen &rsquo; in the code. another quick chat for FE sync or on slack.